openassessment/templates/openassessmentblock/edit/oa_edit_example_based_assessment.html

```diff
@@ -4,19 +4,14 @@
     <div class="drag-handle action"></div>
     <div class="openassessment_inclusion_wrapper">
         <input id="include_ai_assessment" type="checkbox"
-                {% if assessments.example_based_assessment %} checked="true" {% endif %}>
-        <label for="include_ai_assessment">{% trans "Step: Example-Based Assessment" %}</label>
+            {% if assessments.example_based_assessment %} checked="true" {% endif %}>
+        <label for="include_ai_assessment">{% trans "Step: Robotic Assessment" %}</label>
     </div>
     <div class="openassessment_assessment_module_editor">
-        <p id="ai_assessment_description_closed" class="openassessment_description_closed {% if assessments.example_based_assessment %} is--hidden {% endif %}">
-            {% trans "An algorithm assesses learners' responses by comparing the responses to pre-assessed sample responses that the instructor provides."%}
-        </p>
-        <div id="ai_assessment_settings_editor" class="assessment_settings_wrapper {% if not assessments.example_based_assessment %} is--hidden {% endif %}">
+        <div id="ai_assessment_settings_editor" class="assessment_settings_wrapper">
             <p class="openassessment_description">
-                {% trans "Enter one or more sample responses that you've created, and then specify the options that you would choose for each criterion in your rubric. Note that you must add your rubric to the Rubric tab before you can complete this step." %}
-            </p>
-            <textarea id="ai_training_examples">{{ assessments.example_based_assessment.examples }}</textarea>
```

Comment:
I'm going to restore this textarea, I can't figure out how else to expose the example XML I made for my devstack course haha